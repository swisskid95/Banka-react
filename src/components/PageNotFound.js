import React from 'react';
import TopSection from '../components/commons/TopSection';

const PageNotFound = () => (
  <main className="container">
    <div className="main-wrapper">
      <TopSection />
      <h1 className="page-not-found">Oops! Page not found.</h1>
    </div>
  </main>
);

export default PageNotFound;
