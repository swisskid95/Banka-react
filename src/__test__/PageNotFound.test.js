/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import PageNotFound from '../components/PageNotFound';

describe('PageNotFound', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<PageNotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
