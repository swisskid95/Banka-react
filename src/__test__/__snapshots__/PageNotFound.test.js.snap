// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<main
  className="container"
>
  <div
    className="main-wrapper"
  >
    <TopSection />
    <h1
      className="page-not-found"
    >
      Oops! Page not found.
    </h1>
  </div>
</main>
`;
